lib/game_stats.rb

```diff
@@ -1,7 +1,8 @@
-class GameStats
-  attr_reader :games
+require './lib/stats'
+
+class GameStats < Stats
   def initialize(games)
-    @games = games
+    super(games)
```

Comment:
I think I broke it by not calling `super(games, teams, game_teams)` (expected 3 arguments, received 1 error)